horizon:
  problem:
    T_f_horizon: 5.0
    T_takeoff_horizon: 3.0
    T_touchdown_horizon: 3.5
    adaptive_dt: false
    dt: 0.08
    solver:
      name: "ipopt"
      tolerance: 0.0001
      max_iter: 1000
      linear_solver_name: "ma57"

  test_rig:
    lb: -0.4
    ub:  0.4

  hip:
    tau_lim: 50.0

  knee:
    tau_lim: 50.0

  initial_conditions:
    q_p:     [0.0, 0.0, 0.0]
    q_p_dot: [0.0, 0.0, 0.0]

  cost_weights:
    contact_force: 0.01
    terminal_postural: 1000
    small_q_p_ddot: 1
    big_hip_jump: 100



